src/include/daos_pool.h

```diff
@@ -96,23 +96,25 @@ struct daos_rebuild_status {
 	/**
 	 * rebuild is done or not, it is valid only if @rs_version is non-zero
 	 */
-	int32_t			rs_done;
+	int32_t			rs_running;
 
 	/* padding of rebuild status */
 	int32_t			rs_padding32;
 
 	/* Failure on which rank */
 	int32_t			rs_fail_rank;
 	/** # total to-be-rebuilt objects, it's non-zero and increase when
-	 * rebuilding in progress, when rs_done is 1 it will not change anymore
+	 * rebuilding in progress, when rs_running is 1 it will not change anymore
 	 * and should equal to rs_obj_nr. With both rs_toberb_obj_nr and
 	 * rs_obj_nr the user can know the progress of the rebuilding.
 	 */
 	uint64_t		rs_toberb_obj_nr;
-	/** # rebuilt objects, it's non-zero only if rs_done is 1 */
+	/** # rebuilt objects, it's non-zero only if rs_running is 1 */
 	uint64_t		rs_obj_nr;
-	/** # rebuilt records, it's non-zero only if rs_done is 1 */
+	/** # rebuilt records, it's non-zero only if rs_running is 1 */
 	uint64_t		rs_rec_nr;
+	/** # progress of indicator for rebuild in percentage */
+	int32_t			rs_progress;
```

Comment:
Where this is used? I don't see it in this patch.

Reply:
It's currently not used, @skirvan44 requested this be added to the struct. He can provide more input.